prettypandas/styler.py

```diff
@@ -46,8 +47,15 @@ def apply_pretty_globals():
 
 
 Formatter = namedtuple("Formatter", "subset, function")
-
-
+remove_nan_formatter = lambda x: '' if x == np.nan else x
+
+def PrettyPandasNoIndex(dataframe, table_styles = [], *args, **kwargs):
+    return PrettyPandas(dataframe, 
+                        *args, 
+                        table_styles = table_styles + [{'selector': '.row_heading', 'props': [('display', 'none')]}, 
+                                                       {'selector': '.blank', 'props': [('display', 'none')]}],
+                        **kwargs)
+                        
```

Comment:
I'm happy with it being it's own class.

Don't worry about it being dirty, if you want I can go ahead and clean it up for you. It's awesome to get a PR and you've put a lot of good work into this.